app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command):
+    old_file = command.split(" ")[1]
+    new_file = command.split(" ")[2]
+
+    if old_file == new_file:
```

Comment:
also, you should check the input command

Reply:
updated